### PR TITLE
Document the SR v4 Servo Board Hardware

### PR DESCRIPTION
**Pull request by trickeydan**

### CODE REVIEW

File: docs/hardware/srobo/sbv4.rst [hunk 1]
```diff
@@ -1,7 +1,134 @@
...
+
+Controls
+--------
+
+The servo board is largely controlled over the USB interface. There is one physical push button on the board that is a firmware reset button.
+
+USB Interface
+-------------
+
+The Vendor ID is ``1bda`` (University of Southampton) and the product ID is ``0011``.
```
Comment (kierdavis) [inline docs/hardware/srobo/sbv4.rst]:
```suggestion
The vendor ID is ``1bda`` (University of Southampton) and the product ID is ``0011``.
```

Comment (kierdavis) [inline docs/hardware/srobo/sbv4.rst]:
```suggestion
The vendor ID is ``1bda`` (University of Southampton) and the product ID is ``0011``.
```

File: docs/hardware/srobo/sbv4.rst [hunk 2]
```diff
@@ -1,7 +1,134 @@
...
+| Parameter     | Value |
++===============+=======+
+| bmRequestType | 0x00  |
++---------------+-------+
+| bRequest      | 64    |
++---------------+-------+
+
+There are a list of ids defined in the firmware of the servo board that will let you read and write values to it.
+
+It is recommended to read the source to further understand how to control this device.
```
Comment (kierdavis) [inline docs/hardware/srobo/sbv4.rst]:
```suggestion
It is recommended to read the `firmware source`_ to further understand how to control this device.
```

I think this should use the same hyperlink target as the link at the bottom of the page, but I'm not familiar enough with reStructuredText to be sure.

Comment (trickeydan) [reply docs/hardware/srobo/sbv4.rst]: Yeah it should

**Pull request merged**